pyproject.toml

```diff
@@ -25,7 +25,7 @@ maintainers = [
     { name = "Julia Schumann", email = "jschumann@physik.hu-berlin.de" },
 ]
 license = { file = "LICENSE" }
-dependencies = ["nomad-lab>=1.3.0"]
+dependencies = ["nomad-lab>=1.2.1"]
```

Comment:
I had a problem pip installing the plugin, because it complained that it could not find the new nomad-lab version, that's why I downgraded. (but have the newer nomad-lab version locally anyways)

Reply:
Did you provide the --index-url path to the Gitlab registry?

Reply:
The problem is, when using the vs code shortcuts for setting up a venv for a workspace, it automatically tries to install the dependencies, with no option to add the --index-url path, and gives an error

Reply:
but activating the environement manually and providing the --indux-url it works, so I change it back. Hopefully it wont be too long until it will work directly.